server/apps/a12n/serializers.py

```diff
@@ -0,0 +1,112 @@
+from rest_framework import serializers
+from rest_framework_simplejwt.tokens import RefreshToken
+from rest_framework_simplejwt.settings import api_settings
+
+from django.contrib.auth import authenticate
+from django.utils import timezone
+from django.conf import settings
+from django.db.models import Q
+
+import re
+
+from apps.acсount.models import User
```

Comment:
Same, readd [this](https://docs.djangoproject.com/en/5.1/topics/auth/customizing/#referencing-the-user-model).